### PR TITLE
Create importance_of_governance.md

### LINKED ISSUE #41: Issue 06 - Article: Importance of governance
# Request for content

### Request type:
Article

#### Content type:
Opinion Piece

### Description of the requested content
Thoughtful article on the subject `Importance of governance`.
The article should really help the reader grasp how and *why* governance is important in the decentralized context. Why should organizations care about governance?

### Additional information
Minimum 500 words, maximum 1500 words

#### Deadline
By the end of May, in time for Aragon Monthly issue 06

### PR DESCRIPTION
**Pull request by dxiri**
### Closing of a referenced Issue
Closes #41 

### Content category
_Article_ 

#### Content type:
_Opinion Piece_

### Title
The Importance Of Governance

### Description
Article I wrote about the importance of governance, 618 words long. Outlines why governance is important and some issues popular project have faced.

### CODE REVIEW

**Comment by Smokyish** [comment]
@john-light Please review asap.

**Comment by eordano** [comment]
I think the article has a number of points in which it could improve (including typos & grammatical errors).

1. It fails to provide context by just quoting wikipedia and not working around the current state of what we think governance is.
2. It needs to do a more profound work of explaining why is governance important. It has a number of `non-sequitour` fallacies and the language is too lax. I'll take one paragraph as an example of what I think should go deeper:

```
The current problem with this system is that lines are starting to blur.
The Internet has enabled collaboration from individuals residing in different
countries and jurisdictions. These jurisdictions are not always compatible
between each other, which leads to all sorts of issues that affect productivity
between them. If you think about it, those problems are expected since
technology has evolved at a greater pace than governance systems,
which creates a gap between the two.
```

* What are "all sorts of issues"?
* What lines are starting to blur? (Previous paragraph doesn't explain)
* How does technology advancing at a faster rate than technology lead to... a gap between what exactly?

Here are some references that I would love to be discussed in an article like this:

* https://medium.com/@FEhrsam/blockchain-governance-programming-our-future-c3bfe30f2d74
* https://medium.com/@Vlad_Zamfir/against-on-chain-governance-a4ceacd040ca
* https://www.youtube.com/watch?v=w8DjFbCTjus This is perhaps a better definition than wikipedia: "Governance is the process of establishing and maintaining legitimacy of decision making processes"

**Comment by Smokyish** [comment]
Thank you for the great review @eordano! 

@dxiri Please enact the requested changes asap so the article can be considered for merging to Issue 05.

**Comment by dxiri** [comment]
thanks for the review @eordano, will implement the suggestions this week!

File: docs/articles/opinion/importance_of_governance.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+and whether through the laws, norms, power or language of an organized society.
+
+It relates to "the processes of interaction and decision-making among the actors involved in a collective problem that lead to
+the creation, reinforcement, or reproduction of social norms and institutions."
+
+In lay terms, it could be described as the political processes that exist in between formal institutions."
+
+That last line, pretty much sums it up.
+
+If you don't have any governance, you have an anarchy, where there is no direction or common goals, causing confusion and disorder.
```
Comment (john-light) [inline docs/articles/opinion/importance_of_governance.md]: The word "anarchy" mean "no rulers", not "no rules". On the contrary, anarchists have developed highly evolved governance systems.

See:

https://theanarchistlibrary.org/library/the-anarchist-faq-editorial-collective-an-anarchist-faq-02-17#toc2

https://c4ss.org/wp-content/uploads/2014/01/Anarchist-Themes-in-the-Work-of-Elinor-Ostrom.pdf

**Comment by Smokyish** [comment]
Issue 05 has been finished, but unfortunately this article is still missing some requested changes, so moving it to be included for Issue 06.

**Comment by dxiri** [comment]
@eordano @john-light I have implemented some changes according to your comments, please review and advise!

Thanks!

**Comment by john-light** [comment]
>  "Governance is the process of establishing and maintaining the legitimacy of decision-making processes" 

This quote should have attribution for who said it.

File: docs/articles/opinion/importance_of_governance.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+There are multiple projects besides Aragon trying to define new ways of Governance that can endure. [Decred](https://decred.org/), for example,
+aims to solve the problems that Bitcoin has in its governance by implementing a consensus voting model that empowers stakeholders
+and allows for the seamless transition from one set of rules to another. This makes the project move at a faster pace
+than Bitcoin since all stakeholders incentives are aligned with the development of the project itself.
+Governance issues have plagued Bitcoin with multiple forks, community splits, and inability to move forward
+on certain [Bitcoin Improvement Proposals](https://github.com/bitcoin/bips).
+
+Aragon takes governance one step further, by not just allowing voting, but providing a platform to implement whole digital jurisdictions.
+The value of this can't be underestimated, a digital jurisdiction enables all participants to know the rules and automatically (via smart
+contracts) incentivizes good behavior by rewarding it, while punishing lechers or people trying to harm the organization. It also
```
Comment (john-light) [inline docs/articles/opinion/importance_of_governance.md]: > lechers

Sp: leechers

File: docs/articles/opinion/importance_of_governance.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+completely erases the problem with incompatible jurisdictions, since you are now playing by the rules set by a group of people that have
+similar goals, and we all know 2 heads (or a million) are better than 1!
```
Comment (john-light) [inline docs/articles/opinion/importance_of_governance.md]: 2 -> two
1 -> one

https://www.grammarly.com/blog/when-to-spell-out-numbers/

File: docs/articles/opinion/importance_of_governance.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+If you don't behave, you go to jail, get fined and/or barred from certain communities, for example, if you are a
+doctor and kill someone because of negligence, you get expelled from the medic community and forbidden to exercise
+your profession.
+
+Enter governance.
+
+To start, it may be good to define what governance is and then we can drill down into why it is important.
+
+"Governance is the process of establishing and maintaining the legitimacy of decision-making processes"
```
Comment (john-light) [inline docs/articles/opinion/importance_of_governance.md]: This quote should have attribution for who said it.

File: docs/articles/opinion/importance_of_governance.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+Governance issues have plagued Bitcoin with multiple forks, community splits, and inability to move forward
+on certain [Bitcoin Improvement Proposals](https://github.com/bitcoin/bips).
+
+Aragon takes governance one step further, by not just allowing voting, but providing a platform to implement whole digital jurisdictions.
+The value of this can't be underestimated, a digital jurisdiction enables all participants to know the rules and automatically (via smart
+contracts) incentivizes good behavior by rewarding it, while punishing lechers or people trying to harm the organization. It also
+completely erases the problem with incompatible jurisdictions, since you are now playing by the rules set by a group of people that have
+similar goals, and we all know 2 heads (or a million) are better than 1!
+
+[1] http://www.businessinsider.com/r-cuba-inc-open-to-foreign-capitalists-but-within-limits-2015-6?international=true&r=US&IR=T
```
Comment (john-light) [inline docs/articles/opinion/importance_of_governance.md]: The combined use of links and footnotes throughout the article results in inconsistent styling. Use either/or but not both.

**Comment by Smokyish** [comment]
@dxiri Thanks for your submission again! Please review the comments @john-light left for considerations into your future submissions.

Will approve this submission as it is now and merge this article.

**Pull request merged**